server/middleware/setUpAuthentication.ts

```diff
@@ -16,7 +16,7 @@ export default function setUpAuth(): Router {
 
   router.get('/autherror', (req, res) => {
     res.status(401)
-    return res.render('autherror')
+    return res.render('pages/autherror.njk')
```

Comment:
previously we had some pages outside of the pages folder - I've made sure this no longer the case